test/unit/redirector_test.rb

```diff
@@ -6,7 +6,7 @@ class RedirectorTest < ActiveSupport::TestCase
   def app
     Rack::Builder.new do
       use Redirector
-      run ->(env) { [200, { "Content-Type" => "text/html" }, []] }
+      run ->(_) { [200, { "Content-Type" => "text/html" }, []] }
```

Comment:
For this one, you can omit the parentheses (including the underscore) altogether.